src/axom/sidre/core/Buffer.cpp

```diff
@@ -29,7 +29,7 @@ int getValidAllocatorID( int allocID )
 #ifdef AXOM_USE_UMPIRE
```

Comment:
This no longer needs to be guarded by ``AXOM_USE_UMPIRE``, right?

Reply:
Yup. Thanks for catching that. Done.